### ISSUE TITLE
Clicking URLs in attributes of entities doesn't work as expected

### ISSUE THREAD

**Issue by cyberjunky** [home-assistant/frontend#16519]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

URLs in attributes of entities are displayed correctly, but clicking the result on opening a dashboard page instead of following the URL displayed.

They all point to 'http://<hs-url/<dashboardname>/default_view' instead, which is not logical.

![Screenshot from 2023-05-12 17-26-38](https://github.com/home-assistant/frontend/assets/5447161/7e55d05a-3666-4237-bfb1-22746f7a6d3b)

### Describe the behavior you expected

Open the page related to the displayed url instead (like it was before)

### Steps to reproduce the issue

1. Create a entity with a URL as attribute
2. Click on it, or hover over it
3. Notice it points to default_view of dashboard the entity is on.
...

### What version of Home Assistant Core has the issue?

2023.5.2

### What was the last working version of Home Assistant Core?

Unknown

### In which browser are you experiencing the issue with?

Microsoft Edge Microsoft Version 113.0.1774.35 (Official build) (64-bit)

### Which operating system are you using to run this browser?

Ubuntu 23.04

### State of relevant entities

```txt
time received: Fri May 12 17:24:10 2023
group id: 06.015
receivers: Ambulance (001420059)
capcodes: 002029569, 001420999, 001420059
priority: 1
disciplines: Traumaheli LifeLiner 2 - MMT 90.1
raw message: FLEX|2023-05-12 15:24:10|1600/2/K/A|06.015|002029569 001420999 001420059|ALN|A1 AMBU 17992 van Vredenburchweg 2282SH Rijswijk ZH RIJSZH bon 67946
region: Rotterdam-Rijnmond
location: VRR - AZRR
postal code: 2282 SH
city: Rijswijk
address: Vredenburchweg 2282 SH Rijswijk
street: Vredenburchweg
remarks: LifeLiner 2
longitude: 4.3299963
latitude: 52.0535767
opencage: enabled: True ratelimit: False (1375) geocoded: True
mapurl: https://www.openstreetmap.org/?mlat=52.05358&mlon=4.33000#map=17/52.05358/4.33000
distance: 38.42
icon: mdi:helicopter
friendly_name: Lifeliners
```

### Problem-relevant frontend configuration

```yaml
- type: vertical-stack
        cards:
          - type: entities
            entities:
              - entity: sensor.p2000_capcodes
            title: P2000 Meldingen
```

**Comment by ildar170975** [comment]
same?
https://github.com/home-assistant/frontend/issues/16227
https://github.com/home-assistant/frontend/issues/16599

**Comment by dipanghosh** [comment]
Yeah, it is the same issue I reported in #16599 

Fixed in commit #16462 

The commit is still in the dev channel. So maybe once 2023.6 is released, this will be fixed.

Suggest to downpatch to 2023.3.x till that happens.

**Comment by karwosts** [comment]
Closing this as fixed, as it is reported fixed, and it is working for me.
